client/src/components/DirectoryAddUpdate.jsx

```diff
@@ -92,6 +95,76 @@ export default function DirectoryAddUpdate() {
     setErrors();
   }
 
+  const oldValues = {};
+
+  function convertProposedChanges() {
+    if (coopObj.proposed_changes.name) {
+      oldValues.coop_name = coopName;
+      setCoopName(coopObj.proposed_changes.name);
+    }
+    if (coopObj.proposed_changes.email.email) {
+      oldValues.contact_email = contactEmail;
+      setContactEmail(coopObj.proposed_changes.email.email);
+    }
+    if (coopObj.proposed_changes.phone.phone) {
+      oldValues.contact_phone = contactPhone;
+      setContactPhone(coopObj.proposed_changes.phone.phone);
+    }
+    if (coopObj.proposed_changes.types) {
+      oldValues.entity_types = entityTypes;
+      setEntityTypes(coopObj.proposed_changes.types.map(item => item.name));
+    }
+
+    // Should this be set up for multiple sites?
+    if (coopObj.proposed_changes.web_site) {
+      oldValues.websites = websites;
+      setWebsites(coopObj.proposed_changes.web_site);
+    }
+
+    if (coopObj.proposed_changes.coopaddresstags_set[0].address.formatted) {
+      oldValues.street = street;
+      setStreet(coopObj.proposed_changes.coopaddresstags_set[0].address.formatted);
+    }
+    if (coopObj.proposed_changes.coopaddresstags_set[0].address.locality.name) {
+      oldValues.city = city;
+      setCity(coopObj.proposed_changes.coopaddresstags_set[0].address.locality.name);
+    }
+    if (coopObj.proposed_changes.coopaddresstags_set[0].address.locality.state.code) {
+      oldValues.state = state;
+      setState(coopObj.proposed_changes.coopaddresstags_set[0].address.locality.state.code);
+    }
+    if (coopObj.proposed_changes.coopaddresstags_set[0].address.locality.postal_code) {
+      oldValues.zip = zip;
+      setZip(coopObj.proposed_changes.coopaddresstags_set[0].address.locality.postal_code);
+    }
+    if (coopObj.proposed_changes.coopaddresstags_set[0].is_public) {
+      oldValues.address_public = addressPublic;
+      setAddressPublic(coopObj.proposed_changes.coopaddresstags_set[0].is_public);
+    }
+    if (coopObj.proposed_changes.description) {
+      oldValues.description = descEng;
+      setDescEng(coopObj.proposed_changes.description);
+    }
+  }
+
+  function checkExistingEntity() {
+    if (coopObj.proposed_changes) {
+      convertProposedChanges();
+    }
+
+    let formElements = document.querySelectorAll('.form-control');
+
+    formElements.forEach(input => {
+      if (oldValues.hasOwnProperty(input.name)) {
+        input.style.borderColor = 'red';
```

Comment:
Instead of hard-coding the CSS properties here, could we set this elements CSS class and then define that in the CSS file you have below?  I feel like this woudl make things slightly more flexible if Steve or others want different styles to highlight the values.